crates/sui-framework/docs/linked_table.md

```diff
@@ -126,20 +126,20 @@ removal
 ## Constants
 
 
-<a name="0x2_linked_table_ETableNotEmpty"></a>
+<a name="0x2_linked_table_ETableIsEmpty"></a>
```

Comment:
I believe the docs don't currently get wiped so if a file is moved, its docs will be duplicated.  You may want to delete all the docs folders and regenerate them with your new directory structure (unless you haven't already done that).